### DISCUSSION TITLE
ESXi 7.0.0 SO Standalone 2.3.2 Install From ISO Fails with Errors = 0 in sosetup.log

### DISCUSSION THREAD

**Discussion by VDAAustin** [Security-Onion-Solutions/securityonion#1871] [General]
Working on installing Security Onion 2.3.2 in ESXi 7.0.0 using the official ISO. Install works perfectly until the end where it fails after the verification process. This is a standalone install being attempted.

System specs

8 x 2.3Ghz cores
64GB of RAM
3 x 10Gbps VMXNET3 NIC
2TB of Storage

It hangs at verifying setup followed by errors

![image](https://user-images.githubusercontent.com/55412947/98723182-0acd6e80-2360-11eb-8cf2-495214d29c51.png)

Looking through soerror.log shows Errors = 0 at the end but my so-aptcacherrng and so-mysql appear to be missing.

![image](https://user-images.githubusercontent.com/55412947/98723937-048bc200-2361-11eb-83b4-a2ce554d6f14.png)

Looking through so setup.log here is what the end of the log looks like.

![image](https://user-images.githubusercontent.com/55412947/98724125-43ba1300-2361-11eb-91ce-d4cb1597cd36.png)

**Comment by m0duspwnens** [comment]
Hi @VDAAustin 

Regarding the MISSING so-aptcacherng from so-status, during install, did you choose `OPEN` when asked `How would you like to download OS package updates for your grid?` If so, this is a bug with so-status that will be resolved in the next release.

Regarding so-mysql missing, can you search the sosetup.log for `ERROR` and report anything found around the `ERROR` lines?

Has the box been rebooted? Is it still having the so-mysql issue following reboot and allowing for first highstate to run?

**Reply by VDAAustin** [reply]
I sure did use open so that explains that issue with aptcachenrg. 

So here is what I ended up figuring out. 

Using so-mysql-start I was able to bring up mysql so it appears to be a bug in it not starting and not thinking it exists after a install. Manually starting it and waiting fixed made it appear as "OK" and everything came online. 

As for reboot I wasn't patient enough the first time waiting for it to boot so I didn't realize how long services take to come online (I am impatient, this is on me).

This appears to be a bug in first start of mysql after a install only.

It appears the last commands run are so-mysql and not so-mysql-stop. Perhaps that is causing the issue?

**Reply by VDAAustin** [reply]
![image](https://user-images.githubusercontent.com/55412947/98829337-16c03b80-2407-11eb-9561-b963ba0ccdf3.png)
